libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Teams/Schemas/Actions/Teams.SendMessagingExtensionSelectItemResponse.schema

```diff
@@ -2,7 +2,7 @@
     "$schema": "https://schemas.botframework.com/schemas/component/v1.0/component.schema",
     "$role": "implements(Microsoft.IDialog)",
     "title": "Send a messaging extension response when an item is selected",
-    "description": "Query link requests have activity.name='composeExtension/queryLink'.",
+    "description": "Query link requests have activity.name='composeExtension/selectItem'.",
     "type": "object",
```

Comment:
Same comments here--long name and weird description.

Reply:
Fixed description.  Not sure what to do about the name... Definitely open to changing it.  

@clearab any suggestions on naming for this, and SendMessagingExtensionConfigQuerySettingUrlResponse  I've followed the TeamsActivityHandler naming, but it is quite long.

Reply:
Could probably drop some words from there, but the names do map to the existing SDK naming - maybe SendMessagingExtensionConfigUrlResponse ? For the other SendMessagingExtensionSelectResponse.schema

I'm assuming the "SendMessagingExtension<blah>Response" format makes only the "<blah>" portion open for renaming. We could shorten "MessagingExtension" to "ME" in all of them, but that ONLY works if it is always clear from context when this is displayed that you're working with a messaging extension.

Reply:
Triggers and response names have been shortened